### DISCUSSION TITLE
Demo Mode not working stuck in connecting loop INAV Configurator  8.0.0 RC2

### DISCUSSION THREAD

**Discussion by synersignart** [iNavFlight/inav-configurator#2276] [General]
guys on Windows 10 I ran into an minor issue on getting demo mode to work.

**Comment by sensei-hacker** [comment]
The SITL binary, used by demo mode, will be updated for the actual release.

**Reply by synersignart** [reply]
ah okie noted .  hows the INAV 8.0.0 RC2 yet are there inherit issues when tuning the drone im gonna try it out lateron

**Comment by sensei-hacker** [comment]
Approximately all known issues (from RC1) are fixed in RC2.

The known ones. :)

We think RC2 is probably about release quality. We'll have more confidence after it's tested more over the next few weeks.
